src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -692,7 +692,7 @@ private function addServices()
         return $publicServices.$privateServices;
     }
 
-    private function addNewInstance(Definition $definition, $return, $instantiation)
+    private function addNewInstance(Definition $definition, $return, $instantiation, $id)
```

Comment:
It's a private method so I don't think it really matters that much. For the sake of DX, I'd say it's a good case to pass it along because it's required to build the error message.